proto/httpcache.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "grpc.cache_client";
+
+package cache_client;
+
+import "cacheclient.proto";
+import "google/api/httpbody.proto";
+import "google/api/annotations.proto";
+
+service HttpCache {
+  rpc Get (_GetRequest) returns (google.api.HttpBody) {
+    option (google.api.http) = {
+      get: "/cache/get/{cache_key}"
```

Comment:
Thinking about pre-signed URL's I am not sure if query params need to be a part of this? For pre-signed URLs I had imagined this to be "/cache/get?token={token}" where the `cache_key` was a part of the token as we need to sign the key as well

Reply:
Yeah I was going to do that as a second change but I can't come up with any reason not to do it now. Thanks, will update!

I think we need something along the lines of:
`/cache/get/{cache_name}/{cache_key}?token={cache_token}`

Reply:
Thanks!! Do we need the `cache_key` to be a part of the path.

Another question - So for non-presigned URLs where will the usual GRPC headers go (auth and cache name)

Reply:
for non-presigned HTTP interactions they will go to exactly this same path. We're not signing the URL, we're granting access via the token; so a limited token will only allow interactions to a limited key range (e.g., 1 key) and an unlimited token (e.g., our current keys) will allow interactions to the whole cache